packages/beacon-node/src/api/impl/events/index.ts

```diff
@@ -33,35 +35,42 @@ export function getEventsApi({chain, config}: Pick<ApiModules, "chain" | "config
     ) => routes.events.EventData[K][];
   } = {
     [routes.events.EventType.head]: (data) => [data],
-    [routes.events.EventType.block]: (block) => [
-      {
-        block: toHexString(config.getForkTypes(block.message.slot).BeaconBlock.hashTreeRoot(block.message)),
-        slot: block.message.slot,
-        executionOptimistic: IS_OPTIMISTIC_TEMP,
-      },
-    ],
+    [routes.events.EventType.block]: (block) => {
```

Comment:
Same as below, change the ChainEvent.block type to match EventType.block so you don't need to re-compute the root here, nor do forkchoice calls